website_snippet_carousel_product/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 Tecnativa - Alexandre D. Díaz
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Website Snippet Carousel Product",
+    "version": "12.0.1.0.0",
+    "category": "Website",
+    "author": "Tecnativa, " "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/e-commerce",
+    "license": "AGPL-3",
```

Comment:
For website modules, we use LGPL-3 for not forcing themes over this module to be released. Please review all your website modules according this.

```suggestion
    "license": "LGPL-3",
```